node_scenarios/node_scenarios_example.yml

```diff
@@ -0,0 +1,16 @@
+node_scenarios:
+  - actions: 
```

Comment:
It might be nice to add a name to each node scenario here, similar to pod kill scenarios

Reply:
The items under `actions` refer to the node scenarios to be applied on a selected node.

Reply:
Sorry, I meant above that 

node_scenarios:
  - name: Stop and Start Scenario
    actions: 
    - node_stop_scenario
    - node_start_scenario
    node_name:
    label_selector: node-role.kubernetes.io/worker
    instance_kill_count: 1
    timeout: 120
    cloud_type: aws
  - name: Reboot Scenario
    actions:
    - node_reboot_scenario
    node_name:
    label_selector: node-role.kubernetes.io/infra
    instance_kill_count: 1
    timeout: 120
    cloud_type: aws

Reply:
In the above case, if 5 different scenarios have to be applied on the same node one after another, the same configurations have to be specified 5 times which is redundant.

Reply:
I think my location of this comment is making it confusing and I think we are talking about two different things here. I am just suggesting that each node scenario in the list has a name, not each action and no need for redundancy 

Here is an example from the etcd scenario. Here the scenario itself has a name and then you can perform multiple steps/actions
```
scenarios:
  **- name: "delete etcd pods"**
    steps:
    - podAction:
        matches:
          - labels:
              namespace: "openshift-etcd"
              selector: "k8s-app=etcd"
```


Similarly, I am suggesting a name to the top of each node scenario you want to perform on a node. 
```
node_scenarios:
  **- name: Stop and Start Scenario**
    actions: 
    - node_stop_scenario
    - node_start_scenario
    node_name:
    label_selector: node-role.kubernetes.io/worker
    instance_kill_count: 1
    timeout: 120
    cloud_type: aws
```

Reply:
Suppose we want to perform stop, start, terminate, restart actions on node X. The actions would include node_stop_scenario, node_start_scenario, node_terminate_scenario, node_restart_scenario. In this case would the name be Start, Stop, Terminate and Restart scenario?

Reply:
Yeh I see your point now, I agree it's redundant